bin/rm.js

```diff
@@ -23,15 +25,8 @@ const target = process.argv[ 2 ];
 const extensions = process.argv.slice( 3 );
 
 const deleteFolderRecursive = ( path ) => {
-	fs.readdirSync( path ).forEach( ( file ) => {
-		const curPath = path + '/' + file;
-		if( fs.lstatSync( curPath ).isDirectory() ) {
-			deleteFolderRecursive( curPath );
-		} else {
-			fs.unlinkSync( curPath );
-		}
-	} );
-	fs.rmdirSync( path );
+	const rm = 'win32' === os.platform() ? 'rd /S /Q' : 'rm -rf';
+	exec( rm + ' ' + path );
```

Comment:
These arguments need escaping, or they will break for directories with spaces and other special characters.  While this may not be an issue in our current actual usage of this command, I am still -1 on shipping it:

- Other people may copy our code or use it as an example.
- This script may see other uses later on, without the consumer being aware of this caveat.
- It's important to consistently be aware of and avoid issues like this.  In a more important context this would lead to security issues.

You can get around this issue by using [`spawnSync`](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options) instead of `execSync` (needs testing on Windows, but the Node docs claim that arguments are escaped correctly there too).

Finally I would also call the variable `spawnSync` instead of changing the name to `spawn` like it was done here.